terra_layer/models.py

```diff
@@ -13,8 +13,20 @@
 VIEW_CHOICES = [(view['pk'], view['name']) for slug, view in settings.TERRA_LAYER_VIEWS.items()]
 
 
+class Scene(models.Model):
+    name = models.CharField(max_length=255)
+    type = models.CharField(max_length=255, default="map")
+    icon_name = models.CharField(max_length=255, default=None, null=True)
+    icon_path = models.CharField(max_length=255, default=None, null=True)
+
+    class Meta:
+        permissions = (
+            ('can_manage_layers', 'Can manage layers'),
+        )
+
+
 class LayerGroup(models.Model):
-    view = models.IntegerField()
+    view = models.ForeignKey(Scene, on_delete=models.CASCADE, related_name="layerGroups")
```

Comment:
It's not javascript world here, we prefer snake case to camelCase

Reply:
Done